mp/src/game/shared/neo/neo_player_shared.cpp

```diff
@@ -201,3 +204,31 @@ void DMClSortedPlayers(PlayerXPInfo (*pPlayersOrder)[MAX_PLAYERS + 1], int *piTo
 	*piTotalPlayers = iTotalPlayers;
 }
 #endif
+
+void GetClNeoDisplayName(wchar_t (*pWszDisplayName)[NEO_MAX_DISPLAYNAME],
+						 const wchar_t wszNeoName[MAX_PLAYER_NAME_LENGTH + 1],
+						 const wchar_t wszNeoClantag[NEO_MAX_CLANTAG_LENGTH + 1],
+						 const bool bOnlySteamNick)
+{
+	const bool bShowSteamNick = bOnlySteamNick || wszNeoName[0] == '\0';
+	wchar_t wszDisplayName[MAX_PLAYER_NAME_LENGTH + 1] = {};
+	(bShowSteamNick) ? (void)g_pVGuiLocalize->ConvertANSIToUnicode(steamapicontext->SteamFriends()->GetPersonaName(),
+																   wszDisplayName, sizeof(wszDisplayName))
+					 : (void)V_wcscpy_safe(wszDisplayName, wszNeoName);
+
+	(wszNeoClantag[0] != L'\0') ? (void)V_swprintf_safe(*pWszDisplayName, L"[%ls] %ls", wszNeoClantag, wszDisplayName)
+								: (void)V_wcscpy_safe(*pWszDisplayName, wszDisplayName);
+}
+
+void GetClNeoDisplayName(wchar_t (*pWszDisplayName)[NEO_MAX_DISPLAYNAME],
+						 const char *pSzNeoName,
```

Comment:
Ditto on the double-pointer out value being refactorable to just a mutable array